Modules/_asynciomodule.c

```diff
@@ -37,6 +37,8 @@ static PyObject *context_kwname;
 static PyObject *cached_running_holder;
 static volatile uint64_t cached_running_holder_tsid;
 
+/* Counter for autogenerated Task names */
+static unsigned long long task_name_counter = 0;
```

Comment:
Please use `uint64_t`.

Reply:
Got it. AFAIK `unsigned long long` is guaranteed to be large enough to store `uint64_t`. Using the latter makes the code more consistent, so I'd go for it anyways.

You can use this format specifier to format `uint64_t` specifically:

```C
printf("... %" PRIu64 "...", (uint64_t)1);
```